### DISCUSSION TITLE
Would you consider compiling an FEA engine to wasm?

### DISCUSSION THREAD

**Discussion by MattFerraro** [donalffons/opencascade.js#257] [General]
Hello,

I really admire this product and the potential it brings for one day having a great browser-based CAD program. Would you be interested in compiling an FEA engine to wasm so that one day could also have a great browser-based FEA/FEM program? There are many open source options (Nastran-95, mystran, calculix, moose, freeFEM, etc), some in C/C++ and some in Fortran.

If the browser-based CAD program and FEA programs where one program, we'd be most of the way to an open source, browser-based Fusion360!

Cheers,
Matt

**Comment by rifont** [comment]
The WebMYSTRAN project has compiled Mystran to WASM: https://github.com/PR-DC/WebMYSTRAN

Web demo: https://www.webmystran.com/
